tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -237,18 +307,54 @@ def test_cis_complaint():
         defaultPodSecurityPolicyTemplateId=POD_SECURITY_POLICY_TEMPLATE)
     assert cluster.state == "provisioning"
     i = 0
-    for aws_node in aws_nodes:
-        aws_node.execute_command("sudo sysctl -w vm.overcommit_memory=1")
-        aws_node.execute_command("sudo sysctl -w kernel.panic=10")
-        aws_node.execute_command("sudo sysctl -w kernel.panic_on_oops=1")
-        if node_roles[i] == ["etcd"]:
-            aws_node.execute_command("sudo useradd etcd")
-        docker_run_cmd = \
-            get_custom_host_registration_cmd(client, cluster, node_roles[i],
-                                             aws_node)
-        aws_node.execute_command(docker_run_cmd)
-        i += 1
+    if CIS_SCAN_PROFILE == 'rke-cis-1.4':
```

Comment:
Is there some way we can put this code in a common place, so that it can be used across `test_cis_scan.py` and `test_rke_cluster_provisioning.py`?